build.gradle

```diff
@@ -60,8 +54,8 @@ ext.deps = [
     "metrics": "com.codahale.metrics:metrics-core:3.0.2",
     "mockito": "org.mockito:mockito-core:2.23.0",
     "objenesis": "org.objenesis:objenesis:2.6",
-    "play_guice": "com.typesafe.play:play-guice_$scalaVersion:$playVersion",
-    "play_logback": "com.typesafe.play:play-logback_$scalaVersion:$playVersion",
+    //"play_guice": "com.typesafe.play:play-guice_$scalaVersion:$playVersion",
+    //"play_logback": "com.typesafe.play:play-logback_$scalaVersion:$playVersion",
```

Comment:
These modules are only used by the tony-history-server. If you leave them in, do they cause the build to fail?